components/coding/studentPortal/WorkshopsCard.tsx

```diff
@@ -0,0 +1,19 @@
+import { Button } from "../../ui/Button";
+
+// type WorkshopsCardProps = {
+//   title: string;
+// };
```

Comment:
You should add additional properties for video and starterFiles

Video should be a string
And starterFiles should be an optional string

Reply:
https://www.typescriptlang.org/docs/handbook/2/objects.html

Read the part called optional parameters